docs/source/examples/THCHS30.rst

```diff
@@ -1,328 +1,202 @@
 
-.. literalinclude:: ../../../tensorbay/opendataset/THCHS30/loader.py
-   :language: python
-   :name: THCHS30-dataloader
-   :linenos:
-   :emphasize-lines: 13-15,40
+In the dev :ref:`reference/dataset_structure:Segment`,
+there is a sequence of :ref:`reference/dataset_structure:Data`.
+You can get one by index.
 
-There are mainly two steps to write a :ref:`contribution:Dataloader`:
+.. code:: python
 
--  Create a :ref:`basic_concepts:Dataset` and its relevant :ref:`Segments <basic_concepts:Segment>`.
--  Add the :ref:`basic_concepts:Data` and corresponding labels
-   to the created :ref:`Segments <basic_concepts:Segment>`.
+    data = dev_segment[0]
 
-Create Dataset and Segments
----------------------------
+.. note::
 
-Note that after creating the :ref:`basic_concepts:Dataset`,
-you need to load the :ref:`contribution:Catalog`.(L40)
-The catalog file ``catalog.json`` is in the same directory with dataloader file.
+    If the :ref:`reference/dataset_structure:Segment` or
+    :ref:`advanced_features/fusion_dataset/fusion_dataset_structure:fusion segment`
+    is created without given name, then its name will be "".
 
-In this example, there is no ``catalog.json`` file, because the lexion of ``THCHS-30`` is too
-large. The lexion is a list of dictionary lists. Therefore, We need to load subcatalog from the raw
-file and map it to have the catalog.(L57-63)
+In each :ref:`reference/dataset_structure:Data`,
+there is a sequence of :ref:`reference/label_format:Sentence` annotations.
+You can get one by index.
 
-Add Data and Labels
--------------------
+.. code:: python
 
-It takes four key substeps to add the data and the labels:
+    labeled_sentence = data.label.sentence[0]
 
--  Creating :ref:`basic_concepts:Data` and adding content.
--  Creating :ref:`supported_label_types:Sentence` and adding annotations.
--  Appending the Sentence labels to the created Data.
--  Appending the Data to the created Segment.
+There is only one label type in "THCHS-30" dataset, which is ``Sentence``.
 
-See :ref:`this page <supported_label_types:Sentence>` for more details for about Sentence annoation
-details.
+The information stored in  ``spell``, ``sentence`` and ``phone``
+of labeled :ref:`reference/label_format:Sentence`.
 
-.. note::
-   The :ref:`THCHS-30 dataloader <THCHS30-dataloader>` above uses relative import(L13-15).
-   However, when you write your own dataloader you should use regular import as shown below.
-   And when you want to contribute your own dataloader, remember to use relative import.
+you can get details by the related attribute.
 
 .. code:: python
 
-   >>> from graviti.dataset import Data, Dataset
-   >>> from graviti.label import LabeledSentence, SentenceSubcatalog, Word
+    sentence = labeled_sentence.sentence
 
-***********************************
- Upload Local Dataset to TensorBay
-***********************************
+The information stored in ``sentence`` are the arguments of a word object of "Word" list in 
```

Comment:
``spell`` represents the spell within the sentence. See ... for more details.
![image](https://user-images.githubusercontent.com/32992355/111754110-92a92380-88d2-11eb-9544-afd30993b15f.png)